<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qnx::Internal::BlackBerryDeviceConfigurationWidget</class>
 <widget class="QWidget" name="Qnx::Internal::BlackBerryDeviceConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>308</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="hostNameLabel">
     <property name="text">
      <string>&amp;Device name:</string>
     </property>
     <property name="buddy">
      <cstring>hostLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="hostLineEdit">
     <property name="placeholderText">
      <string>IP or host name of the device</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Device &amp;password:</string>
     </property>
     <property name="buddy">
      <cstring>pwdLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="pwdLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showPasswordCheckBox">
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="debugTokenLabel">
     <property name="text">
      <string>Debug token:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>Private key file:</string>
     </property>
     <property name="buddy">
      <cstring>keyFileLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="Utils::PathChooser" name="keyFileLineEdit" native="true"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Connection log:</string>
     </property>
     <property name="buddy">
      <cstring>connectionLog</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="connectionLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="requestButton">
       <property name="text">
        <string>Request</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="debugToken">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>keyFileLineEdit</zorder>
  <zorder>hostNameLabel</zorder>
  <zorder>hostLineEdit</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>keyLabel</zorder>
  <zorder>debugTokenLabel</zorder>
  <zorder>label</zorder>
  <zorder>connectionLog</zorder>
  <zorder>debugToken</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>pwdLineEdit</tabstop>
  <tabstop>showPasswordCheckBox</tabstop>
  <tabstop>requestButton</tabstop>
  <tabstop>uploadButton</tabstop>
  <tabstop>connectionLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
